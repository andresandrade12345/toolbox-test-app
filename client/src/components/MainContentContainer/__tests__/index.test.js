import React from 'react';
import MainContent from '../';

describe('test:MainContent', () => {

    test('renders and pass snapshot test', () => {
        const wrapper = shallow(
            <MainContent />
        );

        return expect(wrapper).toMatchSnapshot();
    });

    test(`contains a div with text 'test'`, () => {
        const wrapper = mount(
            <MainContent>
                <div>Test</div>
            </MainContent>
        );

        const wrapperChildren = shallow(wrapper.prop('children'));

        return expect(wrapperChildren.text()).toBe('Test')
    });

});