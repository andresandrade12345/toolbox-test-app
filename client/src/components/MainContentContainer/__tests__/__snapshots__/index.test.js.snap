// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test:MainContent renders and pass snapshot test 1`] = `
<main>
  <Container
    tag="div"
  />
</main>
`;
